OrbitGl/PresetsDataView.cpp

```diff
@@ -55,17 +101,20 @@ std::string PresetsDataView::GetToolTip(int row, int /*column*/) {
     return OrbitUtils::Compare(presets_[a]->Member, presets_[b]->Member, ascending); \
   }
 
+#define ORBIT_PRESET_FUNC_SORT(Func) \
```

Comment:
This, and also ORBIT_PRESET_SORT, above are only used in one place, which maybe defeats the purpose of defining these macros. It might be better to have the defined only locally where they are used. (Or maybe I'm missing the reason for defining them as a macro here.)

Reply:
Removed both macros, thanks